database/buildSchema/28_review_assignment.sql

```diff
@@ -68,12 +68,12 @@ CREATE TRIGGER review_assignment_trigger
     WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
     EXECUTE FUNCTION public.add_event_to_trigger_queue ();
 
--- TRIGGER (Listener) on review_assignment table: To set status accordingly to assignedSections
+-- TRIGGER (Listener) on review_assignment table: Set assignedSections to Null when changing status to AVAILABLE
 CREATE TRIGGER review_assignment_trigger2
     AFTER UPDATE OF trigger ON public.review_assignment
```

Comment:
```suggestion
    AFTER UPDATE OF status ON public.review_assignment
```

I thought we already fixed this? 🤔